packages/opentelemetry-api/.eslintignore

```diff
@@ -0,0 +1,2 @@
+build
+index.ts
```

Comment:
Why we are ignoring `index.ts`?

Reply:
since the PR, we added some rules and indeed, it's not required anymore